DXMainClient/DXGUI/Generic/PrivacyNotification.cs

```diff
@@ -44,15 +44,29 @@ public override void Initialize()
             lblTermsAndConditions.X = lblMoreInformation.Right + UIDesignConstants.CONTROL_HORIZONTAL_MARGIN;
             lblTermsAndConditions.Y = lblMoreInformation.Y;
             lblTermsAndConditions.Text = "https://cncnet.org/terms-and-conditions";
-            lblTermsAndConditions.LeftClick += (s, e) => Process.Start(lblTermsAndConditions.Text);
+            lblTermsAndConditions.LeftClick += (s, e) =>
+            {
+                using var _ = Process.Start(new ProcessStartInfo
+                {
+                    FileName = lblTermsAndConditions.Text,
+                    UseShellExecute = true
+                });
+            };
```

Comment:
Ugh after seeing this in a lot of files - wouldn't it be better to just make a method that does this exact thing instead of duplicating the code?